templates/banner.ejs

```diff
@@ -3,9 +3,9 @@ var inner = '<div id="agttcnstbnnr"' +
   '<div style="margin: 30px 70px 0;"><span ' +
       'style="font-size: 24px;line-height: 24px;font-weight: 500;color:#76b642">Datenschutzeinwilligung zur ' +
       'Reichweitenmessung</span><br></br>' +
-    '<span>Der Verein Arbeitsgemeinschaft Teletest (kurz AGTT; Details siehe agtt.at/hbb-Messung), deren Mitglied <b><%-CHANNEL_NAME%></b> ist, möchte das Nutzungsverhalten der Zuseher:innen erfassen, um dadurch den Mitgliedern der AGTT die Möglichkeit zu geben, deren TV Angebot stetig verbessern zu können. Dazu benötigt die AGTT Ihre Zustimmung, nach der ein Cookie auf Ihrem Gerät platziert wird, um folgende Informationen auslesen zu können:<br></br>' +
+    '<span>Der Verein Arbeitsgemeinschaft Teletest (kurz AGTT; Details siehe agtt.at/hbb-Messung)<% if(!IS_PRO7){ %>, deren Mitglied <b><%-CHANNEL_NAME%></b> ist<%}%>, möchte das Nutzungsverhalten der Zuseher:innen erfassen, um dadurch den Mitgliedern der AGTT die Möglichkeit zu geben, deren TV Angebot stetig verbessern zu können. Dazu benötigt die AGTT Ihre Zustimmung, nach der ein Cookie auf Ihrem Gerät platziert wird, um folgende Informationen auslesen zu können:<br></br>' +
       'Geräte ID, IP-Adresse, System- bzw. Browserinformationen, Geräteinformationen (verwendet HbbTV-Version, TV-Hersteller, Übertragungsweg via Satellit oder Kabel, Geräteauflösung). ' +
-      'Durch Drücken der blauen Taste gelangen Sie zur Datenschutzerklärung, wo Sie auch den Status Ihrer Einwilligung verwalten können.</span></div>' +
+      'Nähere Informationen zum Datenschutz finden Sie in der HbbTV Applikation des Senders, wo Sie den Status Ihrer Einwilligung verwalten bzw. ändern..</span></div>' +
```

Comment:
```suggestion
      'Nähere Informationen zum Datenschutz finden Sie in der HbbTV Applikation des Senders, wo Sie den Status Ihrer Einwilligung verwalten bzw. ändern können.</span></div>' +
```

Reply:
Just removed the extra dot for now. Will double check with Nico.